pkg/cluster/driver/olvm/apps.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2024, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package olvm
+
+import (
+	"github.com/oracle-cne/ocne/pkg/catalog"
+	"github.com/oracle-cne/ocne/pkg/commands/application/install"
+	"github.com/oracle-cne/ocne/pkg/config/types"
+	"github.com/oracle-cne/ocne/pkg/constants"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+// getApplications gets the applications that are needed for the OLVM provider.
+// These applications will be installed in the bootstrap cluster
+func (cad *OlvmDriver) getApplications() ([]install.ApplicationDescription, error) {
+	proxyValues := map[string]interface{}{
+		"httpsProxy": cad.ClusterConfig.Providers.Olvm.Proxy.HttpsProxy,
+		"httpProxy":  cad.ClusterConfig.Providers.Oci.Proxy.HttpProxy,
+		"noProxy":    cad.ClusterConfig.Providers.Oci.Proxy.NoProxy,
+	}
+
+	return []install.ApplicationDescription{
+		install.ApplicationDescription{
+			Application: &types.Application{
+				Name:      constants.CertManagerChart,
```

Comment:
A similar set of apps is installed for oci cluster api.  Could there be a common set the code shares?

Reply:
Yes there is lots of common code between OCI capi and OLVM capi.  The plan is to refactor when we do the next CAPI provider and have common code that all the providers share.